config/global.ini.php

```diff
@@ -104,6 +104,14 @@
 ; if configured to log in a file, log entries will be made to this file
 logger_file_path = tmp/logs/piwik.log
 
+; if set to 1, will enable monolog's FingersCrossedHandler, which buffers all logs and flushes them if a warning or error
```

Comment:
I wonder if it actually needed to have an option for it or always enable it?

Reply:
ATM I'm not sure what effect this will have in production, thought it could use some more testing before it's enabled by default. I suppose it wouldn't be useful for most users anyway? Unless we create some sort of bug summary feature that grabbed these logs so users could upload these w/ error reports.

Reply:
I reckon it would always be useful to have it enabled so we get detailed logs when we ask users to enable it etc. Could also leave the option, test it for now on production, and follow up to remove the setting in 2 or 3 months and have it always enabled.